test/register-login.spec.js

```diff
@@ -0,0 +1,156 @@
+const { PageFactory } = require("../utils/page_factory");
+const { chromeConfig, sleep, quit } = require("../utils/webdriver/chrome_driver");
+const { expect } = require("chai");
+const { TestData } = require("../utils/test_data/test_data");
+
+describe("Test performs the testing of the login and sign up forms", () => {
+    before(async () => {
+        await chromeConfig();
+    });
+    after(async () => {
+        await sleep();
+        await quit();
+    });
+
+    it("Verify that page is loaded and the title matches the expected", async () => {
+        const homePageTitle = await PageFactory.getPage("Home");
```

Comment:
Please add a separate constant for `await PageFactory.getPage("Home")` expression to avoid code duplicates.